### PR TITLE
Translate 68-95-99.7 rule to spanish

### PR DESCRIPTION
**Pull request by nicoguaro**
## Author: 

- Nicolás Guarín-Zapata

## Language: 

- Spanish

## Terms defined:

-  68_95_997_rule

## Editor

- @ian-flores

## Comments

I am sending a PR with a single term to check that I am doing it correctly. If so, I would probably add others laters. What's a good number, so we don't have too much to review or too many PRs?

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -14,13 +14,21 @@
...
+      aproximadamente el 0,3% de los valores se encuentran más de tres desviaciones
+      estándar por encima o por debajo de la media en la mayoría de los casos.
   fr:
     term: "règle de 68-95-99.7"
     def: >
       Exprime le fait que 68% des valeurs sont localisées à l'intérieur d'un (1)
       [écart-type](#standard_deviation) de la [moyenne](#mean), 95% à l'intérieur de 
-      deux (2) écarts-types, et 99,7% à l'interieur de trois (3) écarts-types. Inversement et dans la plupart des cas, environ 0.3% des valeurs 
+      deux (2) écarts-types, et 99,7% à l'interieur de trois (3) écarts-types.
+      Inversement et dans la plupart des cas, environ 0.3% des valeurs
```
Comment (dpshelio) [inline glossary.yml:31]: I'd suggest to don't change the formatting of other languages (even though I prefer them as you've done it!)

File: glossary.yml [hunk 2]
```diff
@@ -507,6 +525,11 @@
     def: >
       A statistic is biased if it is systematically or consistency different from the
       parameter it is supposed to estimate.
+  es:
+    term: "sesgo"
+    def: >
+      Una estadístico está sesgado si es sistemática o consistentemente diferente
+      del parámetro que se supone que debe estimar.
```
Comment (dpshelio) [inline glossary.yml:532]: Esta frase me suena un poco rara. ¿qué es "una estadísitico"?

**Review by dpshelio — APPROVED** [review]
I've revised it and I have only a small comment, but I approve all the rest! Great job @nicoguaro!!

File: glossary.yml (hunk 1 above)
Comment (nicoguaro) [reply glossary.yml:31]: Yes, I realized this later and didn't change any other empty line.

File: glossary.yml (hunk 2 above)
Comment (nicoguaro) [reply glossary.yml:532]: Lo traduje mal, debía ser:

      Un estadístico está sesgado si es sistemático o consistentemente diferente
      del parámetro que se supone que debe estimar.

**Review by ian-flores — APPROVED** [review]

**Comment by nicoguaro** [comment]
Should I resolve the conflicts?

**Comment by baileythegreen** [comment]
@ian-flores Tagging you because it looks like this might have gotten buried. While you're doing stuff here, you might be able to clear this one up quickly.

**Comment by nicoguaro** [comment]
@dpshelio, there are some conflicts in this PR. Should I resolve them?

**Comment by nicoguaro** [comment]
@ian-flores, do you think you could merge this PR?

**Comment by dpshelio** [comment]
@nicoguaro I think it would be easier for you to resolve these conflicts. I would have to open a PR to you to update yours.

**Comment by nicoguaro** [comment]
> @nicoguaro I think it would be easier for you to resolve these conflicts. I would have to open a PR to you to update yours.

I resolved the conflicts. I tried to not touch what was not in my original PR as much as possible.

**Comment by nicoguaro** [comment]
@dpshelio I think that it is fixed now. It took me a while to figure out a repeated Spanish definition.

**Comment by baileythegreen** [comment]
@nicoguaro @dpshelio 

There doesn't seem to be any reason for this to not be merged at this point. I'm going to try it. Hopefully, the statement about no conflicts is still true.

**Comment by elletjies** [comment]
@all-contributors please add @nicoguaro for translation

**Pull request merged**